samples/src/main/java/common/Utils.java

```diff
@@ -31,4 +32,11 @@ public static String generatePaymentReference() {
     public static String generateOrderId() {
         return RandomStringUtils.randomAlphanumeric(8);
     }
+
+    public static String readInput(LineReader reader, String prompt, String defaultValue) {
+        var userInput = reader.readLine(prompt);
+        if (userInput == null  || userInput.isBlank()) {
+            userInput = defaultValue;
+        } return userInput;
```

Comment:
Np.
Can you put the `return` on the next line please

Reply:
:O That was supposed to be on the next line...